### PR TITLE
Add standalone/jarmod instructions to the Installing Optifine page

### PR DESCRIPTION
**Pull request by Godeps2891**
Preview
image

### CODE REVIEW

File: src/wiki/getting-started/installing-optifine.md [hunk 1]
```diff
@@ -12,7 +12,22 @@ Also, if you're using the Fabric mod loader, there's no native support for it, a
...
+
+This method requires you either have the minecraft version you wish to install OptiFine on top of installed through the vanilla launcher or recreate the folder structure somewhere on your system (/.minecraft/versions/*1.18.2*/*1.18.2*.jar).
+
+Go to <https://optifine.net/downloads> and find your respective Minecraft version (eg. 1.18.2).
+
+Once found, click **mirror**. Now click the **download** button to download your OptiFine jar file.
+
+Execute the OptiFine jar file and change the selected folder location if required, then hit the **"Extract"** button and save the mod file somewhere you'll remember.
+
+Now in the PolyMC launcher edit the instance you wish to install OptiFine on, open the **Version** tab and on the right click **Add to Minecraft.jar**, select the extracted OptiFine mod(the file ending in \_MOD.jar) and confirm.
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:25]:
```suggestion
Now in PolyMC edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted OptiFine jarmod (the file ending in \_MOD.jar) and confirm.
```
image

you don't have to right click

File: src/wiki/getting-started/installing-optifine.md [hunk 2]
```diff
@@ -12,7 +12,22 @@ Also, if you're using the Fabric mod loader, there's no native support for it, a
...
+
+OptiFine will now be copied to and installed as a jarmod for that instance so it is safe to delete the installer and extracted mod.
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:27]:
```suggestion
OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod. 
```

**Review by DioEgizio — CHANGES_REQUESTED** [review]

**Comment by Godeps2891** [comment]
added all suggested changes

**Review by DioEgizio — APPROVED** [review]

File: src/wiki/getting-started/installing-optifine.md [hunk 3]
```diff
@@ -12,7 +12,22 @@ Also, if you're using the Fabric mod loader, there's no native support for it, a
 
 Therefore, you should consider the use of OptiFine alternatives whenever possible. [See the wiki page about it](../install-of-alternatives).
 
-## <img src="https://www.optifine.net/favicon.ico" height="20" /> Installing OptiFine
+## <img src="https://www.optifine.net/favicon.ico" height="20" /> Installing OptiFine standalone
+
+This method requires you either have the minecraft version you wish to install OptiFine on to be installed through the vanilla launcher or to recreate the folder structure somewhere on your system (like /.minecraft/versions/*1.18.2*/*1.18.2*.jar).
```
Comment (Hans5958) [inline src/wiki/getting-started/installing-optifine.md:17]:
```suggestion
This method requires you either have the Minecraft version you wish to install OptiFine on to be installed through the vanilla launcher or to recreate the folder structure somewhere on your system (like /.minecraft/versions/*1.18.2*/*1.18.2*.jar).
```

**Comment by ZekeSmith** [comment]
Looks good to me

**Pull request merged**